examples/preapproval/search/search_subscription_payment_orders.rb

```diff
@@ -12,11 +12,39 @@
 report = PagSeguro::SubscriptionSearchPaymentOrders.new(code, status, options)
 
 if report.valid?
+  puts "====== Report ======"
 
   while report.next_page?
     report.next_page!
 
-    puts report.payment_orders.inspect
+    puts "=> Page #{report.page}/#{report.total_pages}"
+    report.payment_orders.each_with_index do |order, i|
```

Comment:
No man, the letter **i** is the universal symbol to index... Don't you programming in Java or C... `for(int i=0; i <= 10; i++)`

rsrss... I'll modify this.